### DISCUSSION TITLE
mastodon.online - no local or global timeline since sunday

### DISCUSSION THREAD

**Discussion by verrutscht** [mastodon/mastodon#20153] [General]
Hi,
As a user I am reporting that the timelines, local as well as global, have disappeared since approx. sunday. 
This is the case when I access M. either through an Android app or from a browser.
I searched the website for a link to submit a report (or request for help) about this but couldn‘t find any. So I came here.
Anyone can help?
Thanks 
regards
nn

**Comment by connyduck** [comment]
The local and federated timelines have been disabled deliberatly in order to bring server load down a bit.

**Reply by verrutscht** [reply]
ok, thx
